src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -906,6 +907,12 @@ private Conversion DeriveStandardExplicitFromOppositeStandardImplicitConversion(
 
                     break;
 
+                case ConversionKind.ImplicitSpan:
```

Comment:
>ConversionKind.ImplicitSpan [](http://example.com/codeflow?start=21&length=27)

Are we hitting this case? It looks like `ClassifyStandardImplicitConversion(TypeSymbol, ...)` does not return `ConversionKind.ImplicitSpan`.

Reply:
You're right, thanks. I've moved the conversion from "conversion from type" to "conversion from expression" and hence we stopped hitting this case.